subprojects/core/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -204,4 +206,33 @@
      */
     @Incubating
     void buildCache(Action<? super BuildCacheConfiguration> action);
+
+    /**
+     * Adds a action to be called to configure {@code PluginRepositoriesSpec}
+     *
+     * @param pluginSettings The action to execute
+     * @throws IllegalStateException if a plugin has already been applied with the <code>plugins { }</code> block.
+     * @since ??
+     */
+    void pluginRepositories(Action<? super PluginRepositoriesSpec> pluginSettings);
```

Comment:
Should not be here, we only want the new pluginManagement on the `Settings` DSL. The old `pluginRepositories` method should stay on the [InitialPassSettingsScript](https://github.com/gradle/gradle/blob/master/subprojects/core/src/main/java/org/gradle/initialization/InitialPassSettingsScript.java#L47), delegate to plugin management and be deprecated.